vdb/recon/__init__.py

```diff
@@ -68,6 +68,21 @@ def reprargs(trace, fmt, args):
         r.append(rstr)
     return r
 
+def getArgs(trace, args):
+    '''
+    Assuming we are at the instruction after
+    a call, grab the argument at the specified
+    index (skipping the saved instruction pointer).
+    '''
+    cc = trace.getEmulator().getCallingConvention("stdcall")
+    if "64" in trace.getMeta("Architecture"):
+        if trace.getMeta("Platform") == "windows":
+            cc = trace.getEmulator().getCallingConvention("msx64call")
+        else: trace.getEmulator().getCallingConvention("sysvamd64call")
```

Comment:
inline "else: foo" is not style compatible with the rest of the code, i know it's a nit but if we dont try to "dig up" the code base will only become more inconsistent :)